.travis.yml

```diff
@@ -57,53 +59,53 @@ matrix:
 
 cache:
   directories:
-   - $HOME/.composer/cache/files
-   - $HOME/.cache/yarn
-   - $HOME/.phpbrew
+  - $HOME/.composer/cache/files
```

Comment:
I kinda liked these indents!

Reply:
I see they're not consistent throughout the file. If we're going to update these, let's stick to [yaml standards](https://docs.saltstack.com/en/latest/topics/troubleshooting/yaml_idiosyncrasies.html#indentation) which are "two spaces"

Reply:
Fixing! I ended up copy/pasting the file when trying to get past an issue and I think my IDE "fixed" it.